devenv.nix

```diff
@@ -253,14 +259,15 @@ in {
     staticFilePaths = lib.mkOption {
       type = lib.types.str;
       default = "/theme/* /media/* /thumbnail/* /bundles/* /css/* /fonts/* /js/* /recovery/* /sitemap/*";
-      description = "Sets the matcher paths to be \"ignored\" by caddy";
+      description = ''Sets the matcher paths to be "ignored" by caddy'';
     };
   };
 
   config = lib.mkIf cfg.enable {
     packages = [
       pkgs.jq
       pkgs.gnupatch
+      pkgs.gnused
```

Comment:
Is `curl` already included in devenv or should we add it as well?

Reply:
We are using the gnused packages directly, ive removed that one from the package list now.